packages/redwood/src/index.ts

```diff
@@ -150,10 +160,47 @@ export const build: BuildV2 = async ({
   const apiDistPath = join(workPath, 'api', 'dist', 'functions');
   const webDistPath = join(workPath, 'web', 'dist');
   const lambdaOutputs: { [filePath: string]: Lambda } = {};
-  const staticOutputs = await glob('**', webDistPath);
+
+  // Strip out the .html extensions
+  // And populate staticOutputs map with updated paths and contentType
+  const webDistFiles = await glob('**', webDistPath);
+  const staticOutputs: Record<string, FileFsRef> = {};
+
+  for (const [fileName, fileFsRef] of Object.entries(webDistFiles)) {
+    const parsedPath = parsePath(fileFsRef.fsPath);
+
+    if (parsedPath.ext !== '.html') {
+      // No need to transform non-html files
+      staticOutputs[fileName] = fileFsRef;
+    } else {
+      const fileNameWithoutExtension = pathBasename(fileName, '.html');
+
+      const pathWithoutHtmlExtension = join(
+        parsedPath.dir,
+        fileNameWithoutExtension
+      );
+
+      renameSync(fileFsRef.fsPath, pathWithoutHtmlExtension);
```

Comment:
Is there a need to rename the file on the actual filesystem?

Reply:
It shouldn't need to be renamed. The `staticOutputs` should map the name to the file so it can be renamed by changing the key.

Reply:
Misunderstood what Nate suggested. I'll try removing the rename!

Reply:
So... if I remove the rename I'm getting this error

```
🔍  Inspect: https://vercel.com/redwoodjs/prerender-redirects/8Ku5ZWg9hojbdv1JiVDkyfhjf6pP [2s]
Error! ENOENT: no such file or directory, open '/vercel/path1/web/dist/200'
Error! Check your logs at https://prerender-redirects-8406jwmyu-redwoodjs.vercel.app/_logs
```

Welcome to view logs - nothing private in the repo